.pre-commit-config.yaml

```diff
@@ -0,0 +1,26 @@
+repos:
+  - repo: https://github.com/psf/black
+    rev: 24.3.0
+    hooks:
+      - id: black
+        args: ["--target-version", "py38"]
+  - repo: https://github.com/PyCQA/isort
+    rev: 5.13.2
+    hooks:
+      - id: isort
+        args: ["--profile", "black"]
+  - repo: https://github.com/pre-commit/mirrors-prettier
```

Comment:
I haven't typically used prettier for ("pure") python projects, in this case, it only targets markdown and json, right?

Reply:
(Which makes sense, so please keep it!)

Reply:
@andhus This should be using `ruff` which allows you to lint and format:

ruff format .
ruff check .

https://github.com/astral-sh/ruff

Reply:
more the merrier i guess :D 
i've added both ruff and ruff-format hooks, they are fully compatible with flake8